### ISSUE TITLE
Dropbox caching

### ISSUE THREAD

**Issue by jmurff** [SquaredTiki/Dringend-Issues#73]
I am having issues still with dropbox. I add a directory to a folder and I can't get Dringend to see it. I attached images. I fully shutdown Dringend and I have even rebooted the iPad. Just wont see it. I am thinking there is a cache issue because when I bring up the menu it is instantly there and doesn't seem to ping dropbox. Maybe a refresh button to force it to?

-Jim
![desktopdropbox](https://cloud.githubusercontent.com/assets/300393/3062149/8bb3cf94-e217-11e3-873f-8d4d9ddbb961.jpg)
![dringendmenu](https://cloud.githubusercontent.com/assets/300393/3062150/8bb58b4a-e217-11e3-9d7f-a813bce8ec30.PNG)

**Comment by SquaredTiki** [comment]
This is quite the curious issue. I've followed exactly what you've done: left the app open, gone into the project selector beforehand then dismissed it (in case any caching is taking place), created a folder on dropbox, went back in and the newly created folder was there.

Dringend doesn't perform any caching and nor does the Dropbox SDK and even if it did it likely wouldn't be set to persist between the app being closed as well as the device being rebooted.

When you are browsing through folders in the project selector do you see the 'Loading…' row momentarily before the actual items appear? Also was the folder one which was copied in to Dropbox or one you created?

**Comment by jmurff** [comment]
You are correct this is not Dringend. I upgraded to 2.8.2 dropbox on my mac an it horcked up everything. The folder was one I moved... even though Mac shows it as there and synched, i went to dropbox via web and it wasn't there. I let it sit and dropbox finally caught up but I ended up with two folders one in original spot and one in new spot. Also a lot of other stuff i had to clean up. i sent dropbox and email. definitely not your issue. Sorry!
